iui/src/draw/path.rs

```diff
@@ -14,22 +12,37 @@ impl Drop for Path {
     }
 }
 
+#[derive(Clone, Copy, PartialEq)]
+pub enum FillMode {
+    Winding,
+    Alternate,
+}
+
+impl FillMode {
+    fn into_ui_fillmode(self) -> uiDrawFillMode {
+        match self {
+            FillMode::Winding => uiDrawFillModeWinding,
+            FillMode::Alternate => uiDrawFillModeAlternate,
+        }
```

Comment:
Perhaps add a cast here?

```suggestion
        } as uiDrawFillMode
```

This is how I solved the platform-dependent type error in 27312b4